### PR TITLE
Some More Underwear

### PR DESCRIPTION
**Pull request by ParadoxSpace**
An array of fishnet stuff (including sockless ones by Aticius), an alt sports bra, an alt bra (the sprite was unused), lacy bras, a halterneck bra, recolorable binders. Most sprites from Polaris.

### CODE REVIEW

**Comment by ParadoxSpace** [comment]
!review

File: code/datums/underwear/bottom.dm [hunk 1]
```diff
@@ -110,4 +110,8 @@
 /datum/category_item/underwear/bottom/swimbottom
 	name = "Swimming Bottom"
 	icon_state = "swimbottom"
-	has_color = TRUE
\ No newline at end of file
+	has_color = TRUE
+
+/datum/category_item/underwear/bottom/fishnet_lower
+	name = "Fishnets"
+	icon_state = "fishnet_lower"
```
Comment (NonQueueingMatt) [inline code/datums/underwear/bottom.dm]: This is not getting in.

File: code/datums/underwear/socks.dm [hunk 2]
```diff
@@ -182,4 +182,12 @@
...
\ No newline at end of file
+	icon_state = "stockings_yellow"
+
+/datum/category_item/underwear/socks/fishnet
+	name = "Fishnets"
+	icon_state = "fishnet"
+
+/datum/category_item/underwear/socks/fishnet_nosock
+	name = "Fishnets, Short"
+	icon_state = "fishnet_nosock"
```
Comment (NonQueueingMatt) [inline code/datums/underwear/socks.dm]: These are not getting in.

File: code/datums/underwear/bottom.dm (hunk 1 above)
Comment (NonQueueingMatt) [reply code/datums/underwear/bottom.dm]: Nevermind.

File: code/datums/underwear/top.dm [hunk 3]
```diff
@@ -51,3 +78,19 @@
...
+	name = "Fishnet with sleeves"
+	icon_state = "fishnet_sleeves"
+
+/datum/category_item/underwear/top/fishnet_gloves
+	name = "Fishnet with gloves"
+	icon_state = "fishnet_gloves"
+
+/datum/category_item/underwear/top/fishnet_simple
+	name = "Fishnet, simple"
+	icon_state = "fishnet_simple"
```
Comment (NonQueueingMatt) [inline code/datums/underwear/top.dm]: These make no sense. Get rid of them. The leggings are fine.

**Comment by Doxxmedearly** [comment]
Agree with matt. Leggings are fine. Fishnet tops and body outfits are a no-go. Other options are great.

**Comment by alsoandanswer** [comment]
Fishnet stockings are alright to me though

How else am I gonna be a grunge girl?

I'm serious, google it

**Pull request merged**